frontend-project/src/pages/letters/list/index.tsx

```diff
@@ -44,6 +44,11 @@ const TableList: FC<{}> = () => {
       dataIndex: 'modifiedOn',
       render: modifiedOn => modifiedOn.toLocaleString(),
     },
+    {
+      title: formatMessage({ id: 'letters-list.table.columns.attachment.title' }),
```

Comment:
Dobra, tego nie wyłapaliśmy wcześniej, bo to nie było w żaden sposób przypięte do modelu (albo można powiedzieć że relacja jest stworzona odwrotnie) - w każdym razie trzeba to poprawić.

Stworzyłem #456 na to.